### PR TITLE
Password limit

### PR DESCRIPTION
**Pull request by plxity**
I have verified that this pull request:
fixex #1188 
* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

**Comment by plxity** [comment]
@catarak Please review this.

File: client/utils/reduxFormUtils.js [hunk 1]
```diff
@@ -44,7 +44,11 @@ export function validateSettings(formProps) {
   if (formProps.currentPassword && !formProps.newPassword) {
     errors.newPassword = 'Please enter a new password or leave the current password empty.';
   }
-
+  if (formProps.newPassword) {
+    if (formProps.newPassword < 6) {
```
Comment (catarak) [inline client/utils/reduxFormUtils.js]: i think you could combine this into one if statement, and, you're missing .length here:
```js
if (formProps.newPassword && formProps.newPassword.length < 6) {
// ...
}
```

Comment (plxity) [reply client/utils/reduxFormUtils.js]: @catarak I have done the changes, Please review :)

**Comment by plxity** [comment]
@catarak Please review it :)

**Comment by catarak** [comment]
cool, this is working for me!

**Pull request merged**